### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by smithc10**
**What issue does this pull request resolve?**
fixing broken links in docs - https://github.com/ajv-validator/ajv/issues/1873
**What changes did you make?**
changed destination paths in specified md files 
**Is there anything that requires more attention while reviewing?**
No

### CODE REVIEW

**Comment by smithc10** [comment]
Hi all, this is my first open source contribution (of hopefully many)! However, I'm a little stuck on why the coverage report check is failing? 

I ran `test-cov` as specified in the repo and the output shows my changes should't have affected test coverage in any capacity.  I'm not quite sure why 1 lines of markdown would result in this.  Is anyone able to assist a beginner? 

image

**Comment by epoberezkin** [comment]
I wouldn't worry about coverage report failing, some rounding error in coveralls.

File: docs/guide/combining-schemas.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ const ajv = new Ajv()
 const validate = ajv.addSchema(defsSchema).compile(schema)
 ```
 
-See [Options](./api.md#options) and [addSchema](./api.md#add-schema) method.
+See [Options](../options.md) and [addSchema](../api.md###ajv.addSchema) method.
```
Comment (epoberezkin) [inline docs/guide/combining-schemas.md]:
```suggestion
See [Options](../options.md) and [addSchema](../api.md#add-schema) method.
```

shouldn't it be this?

Comment (epoberezkin) [reply docs/guide/combining-schemas.md]: It seems neither works, it probably should be `#ajv-addschema-schema-object-object-key-string-ajv`. It's strange, there was an anchor in markdown, but it seems like vuepress removes it...

Comment (epoberezkin) [reply docs/guide/combining-schemas.md]: Oh, I see, api.md has `<a name="add-schema"></a>` and it has to be `<a id="add-schema"></a>` - could you please update it too in api.md, then my ref would work...

Comment (smithc10) [reply docs/guide/combining-schemas.md]: updated!

Comment (smithc10) [reply docs/guide/combining-schemas.md]: @epoberezkin let me know if any further changes need to be made from my end!

Comment (smithc10) [reply docs/guide/combining-schemas.md]: @epoberezkin bump - just following up on this thank you

**Pull request merged**